datafusion/sqllogictest/test_files/aggregate_skip_partial.slt

```diff
@@ -289,14 +289,8 @@ SELECT c2, approx_median(c5), approx_median(c11) FROM aggregate_test_100 GROUP B
 5 593204320 0.5156586
 
 # Test approx_distinct for varchar / int
-query III
+query error DataFusion error: This feature is not implemented: Support for 'approx_distinct' for data type Utf8View is not implemented
```

Comment:
This would be needed too -- added a note to 
- https://github.com/apache/datafusion/issues/15096